repo/packages/S/spark/12/config.json

```diff
@@ -156,6 +156,67 @@
+                        },
+                        "opt_mesosphere_dir": {
+                            "default": "/opt/mesosphere",
+                            "description": "Volume to mount in docker for modules.",
+                            "type": "string"
+                        },
+                        "dcos_service_account_credential_dir": {
+                            "default": "/run/dcos/etc/mesos",
+                            "description": "Volume to mount in docker for service-account credential.",
+                            "type": "string"
+                        },
+                        "module_json": {
+                            "default": "file:///opt/mesosphere/etc/mesos-scheduler-modules/dcos_authenticatee_module.json",
+                            "description": "JSON file describing the module within the module_dir.",
+                            "type": "string"
+                        },
+                        "module_name": {
+                            "default": "com_mesosphere_dcos_ClassicRPCAuthenticatee",
+                            "description": "Authenticatee module name.",
+                            "type": "string"
+                        },
+                        "dcos_service_account_credential_json": {
+                            "default": "file:///run/dcos/etc/mesos/agent_service_account.json",
+                            "description": "File containing DCOS service-account credential.",
+                            "type": "string"
+                        },
+                        "mesos_credential": {
+                            "default": "{\"principal\": \"mesos_agent\", \"secret\": \"dcos_spark\"}",
+                            "description": "Mesos credentials. The secret is not used.",
+                            "type": "string"
+                        },
+                        "ssl_files_dir": {
+                            "default": "/run/dcos/pki",
+                            "description": "Directory containing SSL-related files.",
+                            "type": "string"
+                         },
+                        "ssl_cert_file": {
+                            "default": "/run/dcos/pki/tls/certs/mesos-slave.crt",
+                            "description": "Path to SSL cert file on the host.",
+                            "type": "string"
+                         },
+                         "ssl_key_file": {
+                              "default": "/run/dcos/pki/tls/private/mesos-slave.key",
+                              "description": "Path to SSL key file on the host.",
+                              "type": "string"
+                         },
+                         "ssl_ca_dir": {
+                              "default": "/run/dcos/pki/certs",
+                              "description": "Path to SSL CA directory on the host.",
+                              "type": "string"
+                         }
```

Comment:
Why are these `ssl_*` parameters under `authn` instead of `ssl`? Is it because they apply to the scheduler->master SSL rather than Spark's own SSL UI/API?